README.md

```diff
@@ -1 +1,45 @@
-# ordinal-scale-stats-py
\ No newline at end of file
+# Ordinal-Scale-Stats-py
+
+Python package that helps you analyze ordinal data.
+
+## Introduction
+
+Ordinal scale data is common. Companies and governments can quickly perform large-scale research with surveys.
```

Comment:
I think it should be,
"Ordinal scale data ARE common"
because word "data" is plural